templates/create-frontend-plugin/skeleton/plugins/${{values.plugin_id}}/package.json

```diff
@@ -17,47 +17,46 @@
   "scripts": {
     "build": "backstage-cli package build",
     "clean": "backstage-cli package clean",
-    "export-dynamic": "janus-cli package export-dynamic-plugin",
```

Comment:
Is it mentioned elsewhere how to run this command?  I guess this is really mostly what makes this a dynamic plugin software template.

Reply:
yes, as we changed the release and build process quite a bit.  And that does maybe suggest that we could also document this software template to just do the plugin exporting via `npx` too rather than having this script entry.